neo/rawio/intanrawio.py

```diff
@@ -838,19 +954,41 @@ def create_one_file_per_signal_dict(dirname):
     "amp",
     "aux",
     "vdd",
-    "board-ANALOG",
+    "board-ANALOG-IN",
```

Comment:
How comes?

Reply:
Because for rhs there is also "board-ANALOG-OUT", but if I split them into separate function it wouldn't matter.

Reply:
Ok, thanks for the explanation!